pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -28,6 +28,7 @@ import (
 	"fmt"
 	"time"
 
+	jsonmerge "github.com/evanphx/json-patch/v5"
 	"gomodules.xyz/jsonpatch/v2"
```

Comment:
should we use a single library? I think, but not certain, json-path can do both functionalities?

Reply:
I'm trying to find a lib that supports both... the gomodules one only supports the rfc6901 type. The evanphx one supports applying rfc6901 patches but doesn't have anything to generate them.

Reply:
Ah, if it can't do both that's fine, just wanted to make sure we were not accidentally using two different ones. I would prefer evanphx where possible since k8s uses it